src/Controls/src/Core/TemplatedView.cs

```diff
@@ -5,7 +5,7 @@
 
 namespace Microsoft.Maui.Controls
 {
-	public class TemplatedView : Layout, IControlTemplated
+	public class TemplatedView : Compatibility.Layout, IControlTemplated
```

Comment:
Just a note for RC, we probably should not have a non compat layout derived from a compat one.

However this should be fine technically since this is just a namespace which is arbitrary.